Bottom: 984069f509ee20549bab107ad5f75b1735584cdf
Top:    5433c6da2185e1949cc787b098ebd18bb2bf5355
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 12:59:28 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index 69de543..081e3cb 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1442,11 +1442,16 @@ int master_show_mounts(struct master *master, const char *maps)
 			if (!me)
 				printf("  no keys found in map\n");
 			else {
+				if (map_name) {
+					write_map(map_name, me);
+					goto next;
+				}
+
 				do {
 					printf("  %s | %s\n", me->key, me->mapent);
 				} while ((me = cache_lookup_next(source->mc, me)));
 			}
-
+next:
 			count++;
 
 			source = source->next;
